mti_nma/steps/nma/nma_viz.py

```diff
@@ -63,7 +67,7 @@ def color_vertices_by_magnitude(
         Filepath to output file of colored mesh object (.blend)
     """
 
-    args = f"-i {path_input_mesh} -o {path_output} -m  {mode} -v {path_vmags}"
+    args = f"-i {path_input_mesh} -o {path_output} -m {mode} -v {path_vmags}"
     cmd = f"{path_blender} -b -P color_vertices -- {args}"
```

Comment:
I think I know why blender was failing with the whole move to `bin` thing. I didn't realize that _Blender_ runs the _Python_. I thought _Python_ ran _Blender_. I _think_ I know how to solve this lets work together later today.

Reply:
Jackson and I talked about this and it looks like by adding `p.stderr` after the `p.subprocess(....)` we could see error output indicating that the problem here is actually that the required data to run this is empty due to an upstream issue. Working on debugging this.

Reply:
For the record: Had to remove p.terminate() after running subprocess because it was prematurely killing blender. This fixed the problem.